dbapi_opentracing/tracing.py

```diff
@@ -43,9 +43,15 @@ def __enter__(self):
         return self.cursor()
 
     def __exit__(self, exc, value, tb):
-        # Pass ConnectionTracing instance to wrapped's __exit__()
-        # for traced commit() and rollback()
-        return self.__wrapped__.__class__.__exit__(self, exc, value, tb)
+        # C extension clients (e.g. psycopg2) require self.__wrapped__.__class__ to be in ConnectionTracing.__bases__,
+        # which wrapt doesn't provide.  We need to trace w/ a best-guess operation here instead of passing self to
+        # self.__wrapped__.__class__.__exit__() as we can w/ pure python clients.
+        operation_name = self._self_rollback_operation_name if exc else self._self_commit_operation_name
+        with self._self_tracer.start_active_span(operation_name) as scope:
+            scope.span.set_tag(tags.DATABASE_TYPE, 'sql')
```

Comment:
Is it possible to make a better guess on the database type based on the connection? I guess I have the same question for line 73 below; ideally we'd like to distinguish between MySQL and PostgreSQL databases.

Reply:
Cool, that's good enough. Can you make sure to document it as best practice?

Reply:
Will do w/ the upcoming version bump